inngest/src/test/kotlin/com/inngest/InngestFunctionConfigBuilderTest.kt

```diff
@@ -85,4 +87,23 @@ class InngestFunctionConfigBuilderTest {
                 .build("app-id", "https://mysite.com/api/inngest")
         }
     }
+
+    @Test
+    fun testCancelOnTimeout() {
+        val durationConfig =
+            InngestFunctionConfigBuilder()
+                .id("test-id")
+                .cancelOn("cancel", null, Duration.ofSeconds(6000))
+                .build("app-id", "https://mysite.com/api/inngest")
+
+        assertEquals<List<Cancellation>?>(listOf(Cancellation("cancel", null, "\"6000s\"")), durationConfig.cancel)
```

Comment:
The klaxon serializer for this adds inner quotes to the value https://github.com/inngest/inngest-kt/blob/bf5da4fe94b802426c8f9c2cba7c19908f32c767/inngest/src/main/kotlin/com/inngest/InngestFunctionConfigBuilder.kt#L263. I tried removing them and several tests broke, so I went with escaping them here.